### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/881f85ceceac17568056d42ba1151e720d9e3ed1

### CODE REVIEW

File: en/firefox/browsers/quantum.ftl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/quantum/
+
+page-title= Download { -brand-name-firefox-quantum }
+page-description= { -brand-name-firefox-quantum } was a revolution. In 2017 we created a new, lightning fast browser that constantly improves. { -brand-name-firefox-quantum } is the { -brand-name-firefox-browser }.
```
Comment (peiying2) [inline en/firefox/browsers/quantum.ftl:8]: Add a comma after `In 2017`

File: en/firefox/browsers/quantum.ftl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+the-latest-firefox = The latest { -brand-name-firefox } engine: { -brand-name-firefox-quantum }
+firefox-quantum-was = { -brand-name-firefox-quantum } was a revolution in { -brand-name-firefox } development. In 2017 we created a new, lightning fast browser that constantly improves. { -brand-name-firefox-quantum } is the { -brand-name-firefox-browser }.
```
Comment (peiying2) [inline en/firefox/browsers/quantum.ftl:11]: Add a comma after `In 2017`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell Minor change requests on punctuation. Otherwise, it looks good.

**Comment by amychurchwell** [comment]
Thanks for the review Peiying! I am going to make these fixes in the PR of an additional page that will be merged soon. :+1:

**Pull request closed without merging**